app/controllers/forest/transactions_controller.rb

```diff
@@ -0,0 +1,9 @@
+class Forest::TransactionsController < ForestLiana::ApplicationController
+
+  def approve_transaction
+    id_transaction = params.dig('data','attributes')["ids"].first.to_i
+    Transaction.update(id_transaction, status: 'validated')
```

Comment:
It's a live demo. So the data must be **read-only**

Reply:
@SeyZ indeed!
As discussed with @louisteenbrink , there are 2 options :
1) we do not show this action & the approval workflow in the live demo. It isweird to show something that you can't actually act on it. This can be quite deceptive.
2) you (because you have writing rights I guess) use the action to create a few approval workflows to populate the collaboration menu, so we can at least show this to clients on a demo. We would need 3 requested (or more), 4 or more to review, and a bit more on History. The numbers are up to you really, just do not put 1 everywhere ;).

I am more a fan of the 2) :D but both options are relevant.